portfolio/src/main/webapp/script.js

```diff
@@ -19,10 +19,15 @@ function addRandomGreeting() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
```

Comment:
If you can remove the unused functionality you probably should, especially if it won't get added back in. Seems like you aren't using the greetings anymore and will probably want to remove them.